.github/workflows/_rocm-test.yml

```diff
@@ -277,3 +277,4 @@ jobs:
 
       - name: Teardown ROCm
         uses: ./.github/actions/teardown-rocm
+        if: always()
```

Comment:
This is needed to ensure that the "Teardown ROCm" step runs even on Cancelled or Failed jobs